src/amber/cli/templates/app/config/application.cr.ecr

```diff
@@ -58,28 +58,33 @@ Amber::Server.configure do |settings|
   #
   # > Read more about Linux PORT REUSE https://lwn.net/Articles/542629/
   #
-  # settings.port_reuse = ENV["PORT_REUSE"]
+  # settings.port_reuse = true
   #
   #
   # Process Count: This will enable Amber to be used in cluster mode,
   # spinning an instance for each number of process specified here.
   # Rule of thumb, always leave at least 1 core available for system processes/resources.
   #
-  # settings.port = (ENV["PORT"] ||= 80).to_i
+  # settings.process_count = ENV["PROCESS_COUNT"].to_i if ENV["PROCESS_COUNT"]?
+  #
+  #
+  # PORT: This is the port that you're application will run on. Examples would be (80, 443, 3000, 8080)
+  #
+  settings.port = ENV["PORT"].to_i if ENV["PORT"]?
```

Comment:
Same question here, should there be a default, or do we want it to fail loudly if that ENV variable isn't there?

Reply:
We're already using `port = 3000` [here](https://github.com/amberframework/amber/blob/c7ffd11d6c65c5347aca41fa0580c7c8d6265e60/src/amber/server/settings.cr#L25)

`:ditto:`